marge/app.py

```diff
@@ -310,7 +310,10 @@ def main(args=None):
             options.ci_timeout = timedelta(minutes=options.max_ci_time_in_minutes)
 
         if options.batch:
-            logging.warning('Experimental batch mode enabled')
+            if options.rebase_remotely:
+                raise Exception("Batch processing does not work together with rebase remotely.")
+            else:
+                logging.warning('Experimental batch mode enabled')
```

Comment:
There might be more cases where `rebase_remotely` does not work. What about everything related to Marge modifying the commit message? :thinking:

Reply:
True... That will very likely not work with the current implementation. :thinking: But you **could** actually set the `merge_commit_message` when merging using the API as well: https://docs.gitlab.com/ee/api/merge_requests.html#merge-a-merge-request

Reply:
Ah, right. Doable but a completely new feature :grin: